src/components/Nav/Nav.scss

```
@@ -0,0 +1,88 @@
+@import '../../styles/Common.scss';
+
+@mixin marginHeaderLeftRight {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.all {
+  width: 1280px;
+  margin: 10px auto;
+
+  .lifeStoreNav {
+    position: absolute;
+    top: 0;
+    width: 1280px;
+    margin: 10px auto;
+
+    .navHeader {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-top: 13px;
+      padding: 10px;
+      color: $--font-desc-gray;
+      font-size: 15px;
+      border-bottom: 1px solid $--border-line;
+
+      .navHeaderLeft {
+        .toMoveNaverPage {
+          @include marginHeaderLeftRight();
+        }
+        .toMoveNaverShopping {
+          @include marginHeaderLeftRight();
+        }
+      }
+      .navHeaderRight {
+        .toLoginPage {
+          @include marginHeaderLeftRight();
```

```suggestion
      .navHeaderLeft {
        .toMoveNaverPage {
          @include marginHeaderLeftRight();
        }
        
        .toMoveNaverShopping {
          @include marginHeaderLeftRight();
        }
      }
      
      .navHeaderRight {
        .toLoginPage {
          @include marginHeaderLeftRight();
```
가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.

---

 넵. 전체적으로 확인 하였으며 수정 하였습니다.